CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Versions
 * Upgrade PHP version: 7.2.0
 * Add rsync to aws image
 * Add zip to aws image
+* Add postgresql-client to php images
```

Comment:
s,images,image, ;)

Reply:
yep I hesitated because they use both syntaxes in the changelog... I chose images cause we create several images from the buimd args. As you wish

Reply:
ha no, I think they made modification to the 5.3 one as well

Reply:
I don't know actually x)

Reply:
Nop only this Dockerfile is enough, thx